### PR TITLE
Other signum CIPs 27, 28, and 29.

### PR DESCRIPTION
**Pull request by jjos2372**

### CODE REVIEW

File: cip-0028.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+```java
+    public long maxSteps(int height) {
+        if(Burst.getFluxCapacitor().getValue(FluxValues.NEXT_FORK, height)) {
+            return 1_000_000L;
+        }
+        return MAX_STEPS.get(atVersion(height));
+    }
+```
+
+## Compatibility
```
Comment (ohager) [inline cip-0028.md:30]: What about the adjustmennt for the `createAT` to allow large SCs? Shouldn't it be mentioned here also?

**Review by ohager — COMMENTED** [review]
lgtm, just my comment about the slightly adjusted createAT API call

File: cip-0028.md (hunk 1 above)
Comment (jjos2372) [reply cip-0028.md:50]: Sure, just did it.

**Review by ohager — APPROVED** [review]
Nice

**Pull request merged**